packages/api/src/routes/blog-subscribe.js

```diff
@@ -0,0 +1,86 @@
+import { JSONResponse } from '../utils/json-response.js'
+import btoa from 'btoa'
```

Comment:
Isn't btoa native - why do we need to import?

Reply:
because there were errors saying it was deprecated. Settled on: Buffer.from(`:${API_KEY}`).toString('base64') with no type errors

Reply:
I can't see any indication that [`btoa`](https://developer.mozilla.org/en-US/docs/Web/API/btoa) is deprecated on MDN.

Personally I'd reach for this over pulling in a Node.js specific library to the bundle and suppress the warning instead.